### PR TITLE
Confidence Gauge -- UI Polish

### PR DESCRIPTION
**Pull request by hannahtwobulls**
Screen Shot 2020-01-06 at 1 49 03 PM

[Live Env](https://nj-career-network-dev5.firebaseapp.com/dashboard/)

[Trello Card](https://trello.com/c/nyyV0prt/71-placeholder-confidence-meter-left-rail-january-polish)

Updated the table styles and typography for the left side rail (Confidence Gauge). Both left side rail (Confidence Level) and right side rail 'Recent Progress' are now are contained by cards. The 'Recent Progress' feed will look temporarily out of whack, but this will be resolved [Story #69](https://trello.com/c/KEBYVILB/69-placeholder-completed-activities-right-rail-january-polish)

### CODE REVIEW

File: components/assessment/Question/SliderQuestion.js [hunk 1]
```diff
@@ -21,7 +21,6 @@ const useStyles = makeStyles(theme => ({
 const height = 6;
 const StyledSlider = withStyles({
   root: {
-    // color: '#52af77',
```
[the remark this answers is not in the document]
Comment (hannahtwobulls) [reply components/assessment/Question/SliderQuestion.js]: done

**Comment by hannahtwobulls** [comment]
> The following is missing:
> 
> * Boarder around entire graphic
> * Dot on speedometer
> 
> image
> 
> See zeplin link for more detail: https://app.zeplin.io/project/5dd4ad65d7803236c37c5c94/screen/5df3fbec8fb6564b1410e2db

Couldn't figure out how to respond to your comment... 

The borders will be addressed in the following PR for the right side rail.
I talked to Crandall about the circle at the point (and if it was completely necessary) and he said it was fine as is.

File: components/assessment/Question/SliderQuestion.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (hannahtwobulls) [reply components/assessment/Question/SliderQuestion.js]: > Can you please also move the `components/dashboard/Dashboard.js` changes to its own PR?

Not sure what you mean by this

[the remark this answers is not in the document]
Comment (rossdakin) [reply components/assessment/Question/SliderQuestion.js]: What I mean is: this PR is meant to address the confidence meter, but it looks like it also includes some incomplete work related to the "recent progress" list:

**In PR**
Screen Shot 2020-01-08 at 10 24 11 AM

**In Zeppelin**
Screen Shot 2020-01-08 at 10 17 43 AM

**Pull request merged**